build-gluster-org/jobs/macros.yml

```diff
@@ -1,13 +1,24 @@
 - scm:
     name: glusterfs
     scm:
+    - git:
+        branches:
+        - ${sha1}
+        refspec: +refs/pull/*:refs/remotes/origin/pr/*
+        url: git://github.com:gluster/glusterfs.git
```

Comment:
we should use https here, cause git: is not encrypted, so we may be attacked with a MITM that would lead to code execution on our builders.